### DISCUSSION TITLE
Merging two repositories

### DISCUSSION THREAD

**Discussion by xXUltrOnionXx** [community/community#23750] [New to GitHub]
<p>Hello all, I am new here on github and I wanted to ask you how can I merge an existing repostery to another axisting repostery ?</p>
<p>I would be realy thankfull if you could help me. Thank you :slight_smile:</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/xxultronionxx">@xXUltrOnionXx</a>, welcome to the community we are happy you are here!</p>
<p>In general, we wouldn’t advise attempting to merge the Git history (that is, the commits) of two repositories. While it is possible, it’s not by any means a standard practice. Indeed, storing a branch within a repository that shares no common commit history with the repository’s other branches is highly unusual and could cause significant confusion.</p>
<p>With regard to issues, we have a documented feature here that would allow you to transfer issues from one repository to the other:</p>
<p><a href="https://help.github.com/en/articles/transferring-an-issue-to-another-repository" class="onebox" target="_blank">https://help.github.com/en/articles/transferring-an-issue-to-another-repository</a></p>
<p>Keep in mind forks, stars, pull requests, and watchers cannot be moved or merged across repositories.</p>
<p>As an alternative to attempting to merge these two repositories, you might consider archiving one of them as a form of deprecation, and adding a link to the current repository within the archived repository’s README file. We have documentation on archiving repositories here:</p>
<p><a href="https://help.github.com/en/articles/archiving-repositories" class="onebox" target="_blank">https://help.github.com/en/articles/archiving-repositories</a></p>
<p>Should you decide to continue with a merge plan, I suggest checking out this very detailed <a href="https://github.community/t/combining-repositories/2060/2">post</a> by Lee, the post has instructions for a trial run and details on the intricacies of merging.</p>
<p>I hope this helps!</p>

**Comment by xXUltrOnionXx** [comment]
<p>That was kind of realy detailed. Thank you very much for helping me :slight_smile:.</p>
<p>So well I will see what I can do then.</p>
<p>Again thank you for helping <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a></p>
